docs/user/03-22-namespace-based-mapping.md

```diff
@@ -1,20 +1,21 @@
-# Create a Service Instance with a Namespace-Based Secret
+# Namespace-Based Mapping
 
 To have service instances from one subaccount associated with one namespace, you must use a Secret dedicated to this namespace to create these service instances.
 
 ## Prerequisites
 
-* A subaccount in the SAP BTP cockpit
+* A subaccount in the SAP BTP cockpit.
+* kubectl configured for communicating with your Kyma instance.
 
 ## Procedure
 
 ### Create a Namespace-Based Secret
```

Comment:
Can we merge the section with the Create Your Custom Secret section. It seems like both do the same but just use different namespaces.

Reply:
It's not exactly the same, and the idea was to describe two complete processes separately.